bitcoin-internals/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "bitcoin-internals"
+version = "0.1.0"
+authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>", "The Rust Bitcoin developers"]
+license = "CC0-1.0"
+repository = "https://github.com/rust-bitcoin/rust-bitcoin/"
+documentation = "https://docs.rs/bitcoin-internals"
+description = "Internal types and macros used by rust-bitcoin"
```

Comment:
Probably put the "ecosystem" or "organization" word at the end to not confuse it with the crate.

Reply:
Used "ecosystem", thanks.